docs/cosmology/units.rst

```diff
@@ -0,0 +1,86 @@
+.. _astropy-cosmology-units-and-equivalencies:
+
+************************************
+Cosmological Units and Equivalencies
+************************************
+
+.. currentmodule:: astropy.cosmology.units
+
+This package defines and collects cosmological units and equivalencies.
+Many of the units and equivalencies are also available in the
+:mod:`astropy.units` namespace.
+
+We suggest importing this units package as
+
+    >>> import astropy.cosmology.units as cu
+
+
+If you want to access all units through :mod:`astropy.units`, then this units
+module can be added to the astropy.units namespace using
+:func:`~astropy.units.add_enabled_units`.
+
+    >>> u.add_enabled_units(cu)  # doctest: +SKIP
+
+
+About the Units
+===============
+
+.. _littleh-and-H0-equivalency:
+
+Reduced Hubble Constant and "little-h" Equivalency
```

Comment:
I agree with https://github.com/astropy/astropy/pull/12092#pullrequestreview-736382855 that it would be better to have just one instance of this text, whether it be here or in the units documentation where it was copied from. This PR hasn't even been merged yet but the copy has already become outdated.

Reply:
Thanks @eerovaher. A good point. I've moved all the definitions to ``cosmology.units``, and the docs as well.
Now only one copy can become outdated!